docs/encoding.md

```diff
@@ -0,0 +1,327 @@
+# Configuring VSCode and PowerShell to prevent encoding bugs
```

Comment:
Does this apply to *both* Windows PowerShell and PowerShell Core?  I thought PowerShell Core was immune from this issue?  If so, maybe this should say "Windows PowerShell"?

Reply:
Nevermind, the above was clarified after reading the rest of the doc.